integration-test/src/main/resources/testsuites/e2e/distrox-encrypt-tests.yaml

```diff
@@ -0,0 +1,5 @@
+name: "distrox-encryption-tests"
```

Comment:
why do we need a different testsuite?

Reply:
This was introduced to can execute the test case separately. However this is not used for E2E or promotion tests. So I'm going to remove. Thanks for pointing out!